uploader.json

```diff
@@ -2179,7 +2179,7 @@
             "symbolic_name": "nginx_conf"
         },
         {
-            "file": "/opt/rh/nginx()*/root/etc/nginx/()*.conf",
+            "file": "/opt/rh/nginx()*/root/etc/nginx/.*.conf",
```

Comment:
This regex will not work because the function will only match against the basename and not the directory.

Reply:
@bfahr, yes, right. so will we update it or just leave it? Because even we update the basename, it can't work.